types/telegram-web-app/index.d.ts

```diff
@@ -135,15 +135,17 @@ interface WebApp {
      * events.
      */
     onEvent(
-        eventType: 'themeChanged' | 'mainButtonClicked' | 'backButtonClicked' | 'settingsButtonClicked' | 'popupClosed',
+        eventType: 'themeChanged' | 'mainButtonClicked' | 'backButtonClicked' | 'settingsButtonClicked',
         eventHandler: () => void,
     ): void;
+    onEvent(eventType: 'popupClosed', eventHandler: (eventData: { button_id?: string }) => void): void;
```

Comment:
According to the docs, this should be `button_id: string | null`, shouldn't it?

> eventHandler receives an object with the single field button_id – the value of the field id of the pressed button. If no buttons were pressed, the field button_id will be null.

Reply:
Ah, yes. Sorry, my typescript skills are not perfect 🙄

Reply:
Please do go ahead and fix it so we can get it merged :)